### ISSUE TITLE
l10n_nl: Company-Contact format

### ISSUE THREAD

**Issue by ploegvde** [odoo/odoo#32109]
Impacted versions: All versions Odoo

Steps to reproduce: Print a sale order or Invoice

Current behavior:
Odoo prints the address header like:
Companyname, contactname

Expected behavior:
In the Netherlands the address format is different then other countries. It is

- When sending to a contact in a company
Companyname
Extra address name (street 2)
T.a.v. : Contactname
Address House Door
ZIP  City
COUNTRY (Not printed on when Netherlands)

- When sending to a person
Companyname
Extra address name (street 2)
Address House Door
ZIP  City
COUNTRY (Not printed on when Netherlands)

Video/Screenshot link (optional):
https://www.postnl.nl/versturen/brief-of-kaart-versturen/hoe-verstuur-ik-een-brief-of-kaart/brief-adresseren/
https://www.postnl.nl/Images/Juiste_adressering_WA-PostNL_tcm10-22447.png
https://www.postnl.nl/Images/juisteadressering2_PostNL_tcm10-22448.gif

**Comment by sswapnesh** [comment]
Is it something related to https://github.com/odoo/odoo/pull/27614?

**Comment by Yenthe666** [comment]
@sswapnesh yes and no. The issue I made is about the address format in the view that is not really correct in the Netherlands. In this case it is about the address on the quotation/order. So kinda related but not the same issue on itself.

**Comment by Yenthe666** [comment]
@mart-e I think we should have a chat about this. This is probably the same case for other countries too. Why don't we make it configurable?

**Comment by C3POdoo** [comment]
Dear @ploegvde,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
